docs/docs/crafty.md

```diff
@@ -3,4 +3,184 @@ id: crafty
 title: Crafty
 ---
 
-Crafty
\ No newline at end of file
+Have you ever wondered what may happen if you combined an Aragon token and fire? A fire-spewing eagle? Burnt chicken? What if Augur's oracles were augmented with Decentraland's tiles, would they start predicting the real estate market? Now, you no longer need to ponder at these vital questions: the Ethereum community will answer them for you!
+
+In [Crafty](https://crafty.zeppelin.solutions), users can create new ERC20 tokens, with a few added goodies (following [EIP-1046](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1046.md)): a picture, and a short description. This allows for each token to have that extra bit of personality that makes something unique. But that's not all: the only way to get these tokens is by crafting them, which requires ingredients (any ERC20!) to be spent. And since the craftable tokens themselves can also be used as ingredients, the posibilities are endless!
+
+## Project setup
+
+This guide will show you the process of making the contracts in the Crafty game upgradeable by using ZeppelinOS, which we'll install globally to have it available on all our projects.
+
+```sh
+npm install --global zos
+```
+
+All snippets here were extracted from the public [Crafty repository](https://github.com/zeppelinos/crafty).
+
+## Contracts
+
+Crafty works with only two contracts:
+* [`Crafty`](https://github.com/zeppelinos/crafty/blob/master/contracts/Crafty.sol) itself, containing the game logic (how new recipes are added, how crafting takes place, etc.) plus some role-based access control ([RBAC](https://github.com/OpenZeppelin/openzeppelin-solidity/blob/zos-release/contracts/ownership/rbac/RBAC.sol), powered by [OpenZeppelin](https://openzeppelin.org/)), for admin tasks, such as deletion of tokens.
```

Comment:
Is the idea of the guide for the user to follow the steps in their own code, or just look? i.e. if the idea is that the user follows, perhaps here you should be clearer bout the setup, and the fact that they would need to create the contracts at this point, install the zeppelin-solidity dependency, etc

Reply:
We opted for this approach since the contracts are longer than Basil's, and we didn't want to include all of the source code in the guide, so it isn't strictly a 'step-by-step': it's more of a showcase of how Crafty was made (and how similar contracts might be adapted to work with zOS). @fiiiu WDYT?

Reply:
I'm ok with not copy-pasting the entire Crafty code here, more on the setup below.